### PR TITLE
Add Croatian language

### PR DESCRIPTION
**Pull request by muzena**

### CODE REVIEW

File: po/hr.po [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+#: src/exm-application.c:107
+msgid "Extension Manager"
+msgstr "Upravitelj proširenja"
+
+#. TRANSLATORS: 'Name <email@domain.com>' or 'Name https://website.example'
+#: src/exm-application.c:117
+msgid "translator-credits"
+msgstr ""
+"Launchpad Contributions:\n"
+"  gogo https://launchpad.net/~trebelnik-stefina"
```
Comment (mjakeman) [inline po/hr.po:148]: In the interest of consistency, could you write this in the format: `Firstname Lastname` or `Firstname Lastname <email@website.com>` please?

**Review by mjakeman — CHANGES_REQUESTED** [review]
Thanks for the contribution!

It all looks good to me, just one minor point about the translator-credits string.

**Comment by muzena** [comment]
@mjakeman 
I now ok?

**Comment by mjakeman** [comment]
Yes, thanks - this looks good to merge now

**Review by mjakeman — APPROVED** [review]

**Pull request merged**